New log file opened at Wed Nov 18 12:27:46 GMT 2015 by 2008@PC-CZC53003FG
Taking next connection to: 54313
New client attempting to connect on port: 54314-54313
  Address = /10.50.15.6:54313
This client on port 54314 has uuid ab89303b-dc74-4fb4-a69c-8f130832b578 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-ab89303b-dc74-4fb4-a69c-8f130832b578
Taking next connection to: 54313
New client attempting to connect on port: 54315-54313
  Address = /10.50.15.6:54313
Handshake exception on socket: 54315
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 54313
New client attempting to connect on port: 54316-54313
  Address = /10.50.15.6:54313
This client on port 54316 has uuid eb80452f-bd84-43cb-aa34-38ee21c7a1a8 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-ab89303b-dc74-4fb4-a69c-8f130832b578, activator-eb80452f-bd84-43cb-aa34-38ee21c7a1a8
Taking next connection to: 54313
New client attempting to connect on port: 54333-54313
  Address = /10.50.15.6:54313
Handshake exception on socket: 54333
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 54313
New client attempting to connect on port: 54334-54313
  Address = /10.50.15.6:54313
This client on port 54334 has uuid 6ce32825-0b65-4505-88b3-f898269ff262 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-ab89303b-dc74-4fb4-a69c-8f130832b578, activator-eb80452f-bd84-43cb-aa34-38ee21c7a1a8, play-fork-run-6ce32825-0b65-4505-88b3-f898269ff262
Taking next connection to: 54313
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54313
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54313
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54313
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54313
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54313
